internal/cli/logs.go

```
@@ -0,0 +1,127 @@
+			time.Sleep(200 * time.Millisecond)
+		}
+	}
+}
+
+// listHookLogs prints the paths of recent per-run hook output logs.
+func listHookLogs() error {
+	dir := logging.HookLogDir()
+	entries, err := os.ReadDir(dir)
+	if err != nil {
+		fmt.Println("no hook logs yet")
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 116) but it returns nil

(nilerr)